parser_library/src/parsing/parser_error_listener_ctx.h

```diff
@@ -31,6 +31,13 @@ class parser_error_listener_ctx : public diagnosable_ctx, public parser_error_li
         : diagnosable_ctx(hlasm_ctx)
         , parser_error_listener_substitution(add_diag_with_ctx, substituted, std::move(provider))
     {}
+    parser_error_listener_ctx(context::hlasm_context& hlasm_ctx,
```

Comment:
Why would I want to use this overload instead of parser_error_listener_substitution? It completely bypasses the added value of context.